game/forms.py

```
@@ -9,6 +10,15 @@ class CustomUserCreationForm(UserCreationForm):
     class Meta(UserCreationForm.Meta):
         fields = UserCreationForm.Meta.fields + ('email',)
 
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+
+        if User.objects.filter(email=email).exists():
+            raise forms.ValidationError(
+                "An account with this email already exists. Please log in instead."
+            )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)